src/src/EmailBundle/Repository/EmailLogRepository.php

```diff
@@ -10,4 +12,34 @@
  */
 class EmailLogRepository extends \Doctrine\ORM\EntityRepository
 {
+    public function isEmailSent(
+        $templateName,
+        $html,
+        $plain,
+        $email,
+        $meta,
+        $emailType)
```

Comment:
tu nas zaujimaju podla mna iba `$templateName`, `$email`, `$emailType`

Reply:
Hej to je mozne 😄. Asi som uz z dneska prepracovany lebo som si myslel, ze vsak ked zmenia template tak nech posle znovu, ale asi to je prave to co sa nema diat 😄